Makefile

```diff
@@ -114,7 +114,7 @@ update_chains_file:
 	@echo "Compressing."
 	@$(compress_cmd)
 	@echo "Moving compressed file into project."
-	@mv scripts/chains.json.zip AlphaWallet/Rpc\ Network/chains.zip
+	@mv scripts/chains.json.zip AlphaWallet/Resources/chains.zip
```

Comment:
Can you make a separate PR to update `chains.zip` (if it has to be updated)? Keep the PR atomic.